e2e-tests/commands/insurance/complete-and-submit-export-value-form.js

```diff
@@ -0,0 +1,17 @@
+import { FIELD_VALUES } from '../../constants';
+import completeExportValueForm from './complete-export-value-form';
+import { submitButton } from '../../pages/shared';
+
+/**
+ * completeAndSubmitExportValueForm
+ * Complete and submit the "Export value" form
+ * @param {Object} Object with flags completing and submitting export value form
+ * - policyType: type of policy
+ * - policyMaximumValue: should submit an application with the maximum value of 500000
+ */
+const completeAndSubmitExportValueForm = ({ policyType = FIELD_VALUES.POLICY_TYPE.SINGLE }) => {
+  completeExportValueForm({ policyType });
```

Comment:
is this worth being a cypress command, then it can be used in the save and back or change e2e tests without submitting? not a blocker

Reply:
oh it is a command, we could just use the cy. and remove the import

Reply:
As discussed will address in the next PR